drivers/display/ssd1306.c

```diff
@@ -364,14 +364,14 @@ static int ssd1306_init_device(struct device *dev)
 	};
 
 #ifdef DT_INST_0_SOLOMON_SSD1306FB_RESET_GPIOS_CONTROLLER
-	gpio_pin_write(driver->reset,
-		       DT_INST_0_SOLOMON_SSD1306FB_RESET_GPIOS_PIN, 1);
+	gpio_pin_set(driver->reset,
+		     DT_INST_0_SOLOMON_SSD1306FB_RESET_GPIOS_PIN, 0);
```

Comment:
Upon initialization the pin will be set to inactive, there is no need to do it again, the call can be removed.